server/src/main/java/io/spine/server/stand/EventUpdateHandler.java

```diff
@@ -20,27 +20,66 @@
 
 package io.spine.server.stand;
 
+import com.google.protobuf.Any;
+import com.google.protobuf.Message;
+import io.spine.base.Identifier;
 import io.spine.client.EventUpdates;
 import io.spine.client.Subscription;
 import io.spine.client.SubscriptionUpdate;
+import io.spine.client.TargetFilters;
 import io.spine.core.Event;
+import io.spine.core.EventId;
 import io.spine.core.Responses;
 import io.spine.server.type.EventEnvelope;
+import io.spine.type.TypeUrl;
+
+import java.util.Optional;
 
 /**
- * Updates an event subscription based on the incoming event.
+ * The update handler of {@code Subscription}s for {@code Event}s.
  */
-final class EventSubscriptionCallback extends SubscriptionCallback {
+class EventUpdateHandler extends UpdateHandler {
```

Comment:
It used to be `final`, and I cannot find classes derived from it. Can we make it `final` too?